torba/torba/client/baseledger.py

```diff
@@ -422,8 +422,9 @@ def process_status_update(self, update):
                         check_db_for_txos.append(txi.txo_ref.id)
 
                 referenced_txos = {
-                    txo.id: txo for txo in await self.db.get_txos(txoid__in=check_db_for_txos)
-                }
+                    txo.id: txo for txo in await self.db.get_txos(
+                        my_accounts=None, txoid__in=check_db_for_txos
+                    )}
```

Comment:
`my_accounts` used to default to `None` in `get_txos()`, can it just go back to defaulting to `None`? Then this change will no longer be necessary.